src/sentry/static/sentry/app/views/substances/substances.jsx

```diff
@@ -82,36 +83,49 @@ class Substances extends React.Component {
       {
         Header: 'Container',
         id: 'container',
-        accessor: d => (d.location ? d.location.container.name : '<No location>'),
```

Comment:
I hadn't thought much about how headers should be handled in this case, but I have some ideas now. I think it might be nice if we have two header definitions. I think it will be best to take that discussion on a whiteboard meeting.

Reply:
Yes, I had the same idea.